acceptance/detector_test.go

```diff
@@ -316,151 +316,6 @@ func testDetector(t *testing.T, when spec.G, it spec.S) {
-						"--volume", expectedOrderTOMLPath+":/cnb/order.toml",
-						"--env", "CNB_ORDER_PATH=",
-						"--env", "CNB_PLATFORM_API="+latestPlatformAPI,
-					),
-					h.WithArgs("-log-level=debug"),
-				)
-
-				// check group.toml
-				foundGroupTOML := filepath.Join(copyDir, "layers", "group.toml")
-				var buildpackGroup buildpack.Group
-				_, err := toml.DecodeFile(foundGroupTOML, &buildpackGroup)
-				h.AssertNil(t, err)
-				h.AssertEq(t, buildpackGroup.Group[0].ID, "simple_buildpack")
-				h.AssertEq(t, buildpackGroup.Group[0].Version, "simple_buildpack_version")
-			})
-		})
-
-		when("only /layers/order.toml is present", func() {
-			it("processes /layers/order.toml", func() {
-				h.DockerRunAndCopy(t,
-					containerName,
-					copyDir,
-					"/layers",
-					detectImage,
-					h.WithFlags("--user", userID,
-						"--volume", expectedOrderTOMLPath+":/layers/order.toml",
-						"--env", "CNB_ORDER_PATH=",
-						"--env", "CNB_PLATFORM_API="+latestPlatformAPI,
-					),
-					h.WithArgs("-log-level=debug"),
-				)
-
-				// check group.toml
-				foundGroupTOML := filepath.Join(copyDir, "layers", "group.toml")
-				var buildpackGroup buildpack.Group
-				_, err := toml.DecodeFile(foundGroupTOML, &buildpackGroup)
-				h.AssertNil(t, err)
-				h.AssertEq(t, buildpackGroup.Group[0].ID, "simple_buildpack")
-				h.AssertEq(t, buildpackGroup.Group[0].Version, "simple_buildpack_version")
-			})
-		})
-
-		when("platform api < 0.6", func() {
-			when("/cnb/order.toml and /layers/order.toml are present", func() {
-				it("only processes /cnb/order.toml", func() {
-					h.DockerRunAndCopy(t,
-						containerName,
-						copyDir,
-						"/layers",
-						detectImage,
-						h.WithFlags("--user", userID,
-							"--volume", expectedOrderTOMLPath+":/cnb/order.toml",
-							"--volume", otherOrderTOMLPath+":/layers/order.toml",
-							"--env", "CNB_PLATFORM_API=0.5",
-							"--env", "CNB_ORDER_PATH=",
-						),
-						h.WithArgs("-log-level=debug"),
-					)
-
-					// check group.toml
-					foundGroupTOML := filepath.Join(copyDir, "layers", "group.toml")
-					var buildpackGroup buildpack.Group
-					_, err := toml.DecodeFile(foundGroupTOML, &buildpackGroup)
-					h.AssertNil(t, err)
-					h.AssertEq(t, buildpackGroup.Group[0].ID, "simple_buildpack")
-					h.AssertEq(t, buildpackGroup.Group[0].Version, "simple_buildpack_version")
-				})
-			})
-
-			when("only /layers/order.toml is present", func() {
-				it("errors", func() {
-					command := exec.Command("docker", "run",
-						"--user", userID,
-						"--volume", otherOrderTOMLPath+":/layers/order.toml",
-						"--env", "CNB_PLATFORM_API=0.5",
-						"--env", "CNB_ORDER_PATH=",
-						"--rm", detectImage)
-					output, err := command.CombinedOutput()
-					h.AssertNotNil(t, err)
-					expected := "failed to initialize detector: reading buildpack order file: open /cnb/order.toml: no such file or directory"
-					h.AssertStringContains(t, string(output), expected)
-				})
-			})
```

Comment:
Not explicitly covered in detect_inputs_test.go, but this test was kind of redundant IMO.